src/System.Management.Automation/engine/CommandCompletion/ArgumentCompleters.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
Maybe  a name of the file should be `ArgumentCompleter.cs` ?

Reply:
I named it ArgumentCompleters.cs in case we want to consolidate completer classes in one place going forward. Does that sound like a good idea to you? Might make it easier to find them 😂

Reply:
Ah, I meant ScopeArgumentCompleter.cs